### PR TITLE
euse: add page

### PR DESCRIPTION
**Pull request by koknese**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known): 0.6.8**

### CODE REVIEW

File: pages/linux/euse.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# euse
+
+> Gentoo USE flag editor.
```
Comment (acuteenvy) [inline pages/linux/euse.md:3]:
```suggestion
> Enable, disable and obtain information about Gentoo USE flags.
```

File: pages/linux/euse.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+> More information: <https://wiki.gentoo.org/wiki/Euse>.
+
```
Comment (acuteenvy) [inline pages/linux/euse.md:5]: Some more examples from the wiki:
```suggestion

- List active global USE flags:

`euse --active --global`

- List active local USE flags:

`euse --active --local`

```

File: pages/linux/euse.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+- Enable a global USE flag:
+
+`sudo euse --enable {{use_flag}}`
+
+- Disable a global USE flag (Puts a '-' sign in front of the USE flag):
```
Comment (acuteenvy) [inline pages/linux/euse.md:10]:
```suggestion
- Disable a global USE flag (put a '-' sign in front of the USE flag):
```

File: pages/linux/euse.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# euse
+
+> Enable, disable and obtain information about Gentoo USE flags.
```
Comment (spageektti) [inline pages/linux/euse.md:3]:
```suggestion
> Enable, disable, and obtain information about Gentoo USE flags.
```

**Review by gutjuri — APPROVED** [review]
Thanks!

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**